neurolib/models/multimodel/builder/base/backend.py

```diff
@@ -405,18 +507,24 @@ class BackendIntegrator:
         "initialised",
     ]
 
-    def _init_backend(self, backend):
-        if backend == "jitcdde":
-            self.backend_instance = JitcddeBackend()
-        elif backend == "numba":
-            self.backend_instance = NumbaBackend()
-        else:
-            raise ValueError(f"Unknown backend: {backend}")
-        # copy necessary attributes (i.e. symbolic derivatives, helpers, etc) to
-        # backend instance
+    def _init_jitcdde_backend(self):
```

Comment:
instead of `_init_backend` we have now two explicit functions: one for `jitcdde` and one for `numba` backend